about/example_pr/index.md

```diff
@@ -1,3 +1,19 @@
-# Example PR to PyMC
+# Example Pull Request to PyMC
 
-% TODO: make nice and add video
+## Event
+In this video, Reshama does a step-by-step example of how to set up a virtual environment, install the dependencies, build PyMC from source, make a small documentation change, and submit a pull request.
+
+## Details
+
+- Audience: People who are interested in contributing to PyMC and who would like to see an example of a contribution.
```

Comment:
I would also add a link to the docstring guide which is followed during the video. We could also move that page here as "webinar material" like some other talks have. cc @symeneses.

related to discussion in https://github.com/pymc-devs/pymc-data-umbrella/pull/47#discussion_r898249593

Reply:
Note: we can add a `ref` type link here pointing to the `docstring_tutorial` target which will generate a working link already and won't need to be modified if we eventually move that page to a different location.

Reply:
@OriolAbril Are you able to add that info here with suggested changes?

Reply:
added

Reply:
Thank you, I am not so familiar with sphinx, but slowly hoping to learn :)

Reply:
If that looks ok feel free to merge the PR, or update the text shown in the button first then merge: preview at https://pymc-data-umbrella--57.org.readthedocs.build/en/57/about/example_pr/index.html